script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
+
+var a = 10;
+
+const value = 10;
+value++;
+
+//comment whitout space
+
+const func = function(){
+
+  let ad = 10
+  var q=45
+
+}
+
+
+const funcName = ()=>{
+  let varible=10;
+  varible+10
+}
+
+import {var1,var3} from 'module1';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['var3' is defined but never used.](https://app.codacy.com/gh/WebHeroSchool/js_test/pullRequest?prid=5502397)